packages/types/src/block-kit/blocks.ts

```diff
@@ -24,9 +24,11 @@ import {
   RichTextQuote,
   RichTextPreformatted,
   RichTextInput,
+  UrlImageObject,
+  SlackFileImageObject
 } from './block-elements';
 
-export interface Block {
+export type Block = {
```

Comment:
I don't think we can make this change as then `Block` is not allowed to have additional properties beyond what is defined.

Can we leave this as an interface?